client/components/ReviewRating/ReviewList/StarRating.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { AiTwotoneStar, AiOutlineStar } from 'react-icons/ai';
+
+const StarRating = () => (
```

Comment:
This looks interesting 👍 . I found this [JS Fiddle](https://jsfiddle.net/vco9r2rt/3/) that allows you to set a percentage of star width for quarter stars. Can this be used in this implementation?

Reply:
I was looking at the same one, but these pieces codes were part of the commit. I was testing the stars. I will delete it. and it looks like the way to implement our stars.

Reply:
are we going to delete this file now that we have a working star component?